### ISSUE TITLE
an error when building the pro

### ISSUE THREAD

**Issue by toBeStronger-ys** [fralx/LimeReport#252]
Hi!
An error occured when i build the pro limereport,just like the picture below shows.I have tried several times,but never succeed...It confused me a lot,please give me some solutions when you are free.Thanks a lot!
![SFK_6Q`}3}YCDTL7OP4{TP](https://user-images.githubusercontent.com/58078772/75139316-45866c80-5727-11ea-9939-980d3e1403bd.png)

**Comment by fralx** [comment]
Hi!
Do you have an "include" subfolder in your limereport project folder?

**Comment by toBeStronger-ys** [comment]
Yes,And the picture below is the files in the "include" folder.
![MK% O}XU}XD9ITSJW(TTE5J](https://user-images.githubusercontent.com/58078772/75244010-eb100d80-5805-11ea-939a-ccebeaba4341.png)

**Comment by fralx** [comment]
Your first screenshot was created in a time when you build the project from this folder?

**Comment by toBeStronger-ys** [comment]
Yes

**Comment by fralx** [comment]
Look, at first screenshot, the build system tries to create a file in the D:\LimeReport-master\ on the second you show me C:\Users\11039\....

**Comment by toBeStronger-ys** [comment]
Sorry,it is my error.The thing goes like this.First time ,i build the pro at D:/,but i failded and got the error showed in the first screenshot.Then,i copy the whole folder to the desktop,and tried again,but still got the same error.

**Comment by fralx** [comment]
Sorry, something wrong was with Github 
Lets put the project folder on the disk D:
And try again :).

**Comment by toBeStronger-ys** [comment]
emmmm...I have tried several times,but still got the same error.Whatever,thanks for your timely reply.

**Comment by fralx** [comment]
Hi!
As I can see you use Windows and I am confused that "cp" is used. 
If you add QMAKE_COPY = copy /y  to project_folder/limereport/limereport.pro will get some another result?

**Comment by toBeStronger-ys** [comment]
Nope....

**Comment by fralx** [comment]
Could you show me the error message?

**Comment by toBeStronger-ys** [comment]
I add the sentence here. 
![image](https://user-images.githubusercontent.com/58078772/75327544-8ad9a400-58b7-11ea-8863-baea316083ad.png)
and get the same error.Whether it is because i add the sentence at wrong place?
![image](https://user-images.githubusercontent.com/58078772/75328268-b610c300-58b8-11ea-9ff2-459af966adc7.png)

**Comment by fralx** [comment]
You have to modify limereport.pro in the subfolder limereport. 
[limereport.zip](https://github.com/fralx/LimeReport/files/4254893/limereport.zip)

**Comment by toBeStronger-ys** [comment]
Yes,i did.Just like the screenshot shows D:\LimeReport-master\limerreport\limereport.pro

**Comment by fralx** [comment]
Sorry :) Put  QMAKE_COPY = copy /y at the begin of pro file

**Comment by toBeStronger-ys** [comment]
I put the sentence here
![B$T)I7A4ZMNU`FBS7@J2 ZO](https://user-images.githubusercontent.com/58078772/75342116-bd43cb00-58d0-11ea-819c-0a93bb7f2e17.png)
and get the error 
![PCPBF4U0~6HRCT`_BZUVBJ4](https://user-images.githubusercontent.com/58078772/75342480-7d311800-58d1-11ea-8b1d-6d83deb22a99.png)

**Comment by liufeijin** [comment]
Hi   fralx
Many be there is any bug in pro. 
I get some build error also , when i use a static QT5.12.6 to build all limereport project .
The lib only correctly . Other (demo_r1 demo_r2 console designer )can't be created .
![21](https://user-images.githubusercontent.com/19425403/75342121-bf0d8e80-58d0-11ea-9aa0-43aaac3909c0.png)
![1](https://user-images.githubusercontent.com/19425403/75342767-06484f00-58d2-11ea-837a-490bb6653fbc.png)

**Comment by fralx** [comment]
@liufeijin It is not the same error :) 
@toBeStronger-ys  This error looks like the build system is uses Linux command-line tools to manipulate files.
I think it may be because the MinGW compiler environment is accessible by PATH
I need some time to check this theory

**Comment by fralx** [comment]
@toBeStronger-ys  do you use MinGW from Qt installation or you installed it separately?

**Comment by toBeStronger-ys** [comment]
I remember it is from qt installation

**Comment by fralx** [comment]
Maybe GitBush has been installed?

**Comment by toBeStronger-ys** [comment]
Yes! I installed the gitbash

**Comment by fralx** [comment]
Is the GitBush environment accessible by the system environment variable PATH?

**Comment by fralx** [comment]
Could you run GitBush and try to run the command  that you can see in Qt creator something like "cp D:\LimeReport-mastre\ ... ..."

**Comment by toBeStronger-ys** [comment]
sorry, what I installed is gitbash,i don't know whether it is the GitBush you want.
What's more, i also have no idea about what is  the correct command, so i run the following commands, and the last command works.
![2~K4B77DQD7Z~}OJ 1N25$0](https://user-images.githubusercontent.com/58078772/75361018-9944b180-58f1-11ea-8732-5a0a574199e4.png)

**Comment by fralx** [comment]
If you run the last command in cmd window will it work?

**Comment by toBeStronger-ys** [comment]
Sorry to reply you so late.It was nearly 12:00 a.m at that time yesterday in China,so i went to bed. 
As soon as i get up,i try it it at once,and get the same error message showned in the QT Creator.
![BY3~F~V ZJQS$K80~RDFG3Q](https://user-images.githubusercontent.com/58078772/75407810-e2771e80-594e-11ea-8647-af1435f1542b.png)

**Comment by fralx** [comment]
I think the simplest way to build limereport dll in your case is - just comment 36-43 lines
```
    $$PWD/lrglobal.h \
    $$PWD/lrdatasourceintf.h \
    $$PWD/lrdatasourcemanagerintf.h \
    $$PWD/lrreportengine.h \
    $$PWD/lrscriptenginemanagerintf.h \
    $$PWD/lrcallbackdatasourceintf.h \
    $$PWD/lrpreviewreportwidget.h \
    $$PWD/lrreportdesignwindowintrerface.h
```
in limereport.pro and try to build again. 
Then you can look at the PATH variable and find the path that makes an accessible Linux environment on your system.
